src/sentry/filters/web_crawlers.py

```diff
@@ -12,12 +12,12 @@
 CRAWLERS = re.compile(
     r'|'.join(
         (
-            # various Google services
-            r'AdsBot',
-            # Google Adsense
-            r'Mediapartners',
-            # Google+ and Google web search, but not apis-google
-            r'(?<!APIs-)Google',
+            # Google spiders (Adsense and others)
+            # https://support.google.com/webmasters/answer/1061943?hl=en
+            r'Mediapartners-Google',
```

Comment:
Escape these `-` characters since this is a regular expression.

So `r'Mediapartners\-Google'` etc

Reply:
Oh, good call, Matt. Done.